RENotebookApi/Lib.fs

```diff
@@ -17,8 +17,14 @@ type MSAGL =
 
         let geometryGraph = graph.GeometryGraph
 
-        //The SVG writer hardcodes Arial 16.0
-        let font = SixLabors.Fonts.SystemFonts.CreateFont("Arial", 16.f)
+        let font_family = ref null
+        if not (SixLabors.Fonts.SystemFonts.TryFind("Arial", font_family)) then
```

Comment:
This was hardcoded to Arial as MSAGL is effectively internally as well when we do an SVG output. Do the MSAGL outputs  look with the fallbacks?